from context import xyy

def test_fail(): assert False, 'x_x'
